src/data/mod.rs

```diff
@@ -14,43 +16,78 @@ pub struct Context {
     pub repo: GithubRepo,
     pub releases: Vec<Release>,
     pub has_prereleases: bool,
-    pub latest_dist_release: Option<DistRelease>,
+    pub latest_release: Option<usize>,
```

Comment:
i am surprised this is a type usize...

Reply:
it's an index into an array

Reply:
oooooooooooohhhhh- thoughts on call it latest_release_index?